app/javascript/components/Admin/Users/PermissionRisk/Guids.tsx

```diff
@@ -16,7 +16,7 @@ type GuidProps = {
 const Guid = ({ guid, user_ids }: GuidProps) => (
   <div>
     <h5>
-      <Link href={Routes.admin_compliance_guids_path(guid)}>{guid}</Link>
+      <Link href={Routes.admin_guid_path(guid)}>{guid}</Link>
```

Comment:
This is broken in `main`; the section links to the JSON endpoint rather than the GUID show page